models/_trino_upgrade/trino_upgrade_alienswap_base_base_trades.sql

```diff
@@ -45,6 +45,3 @@ FROM {{ source('alienswap_base','Alienswap_evt_OrderFulfilled') }} s
     WHERE s.evt_block_time >= timestamp '{{alienswap_usage_start_date}}'
     {% endif %}
 )
-
```

Comment:
afaik the upgrade still hasn't succeeded so they might still be used to test?

Reply:
i left them around for flo to test, but since we think we know the bug now, we can likely remove. i'd recommend doing in it's own PR though, so we can simply revert the original commit to main. then we'd also have easy option to add back if flo wanted to test again